### ISSUE TITLE
ML 1.13 : Types of Distribution (A)

### ISSUE THREAD

**Issue by Isha307** [girlscript/winter-of-contributing#1963]
### Description

Welcome to 'ML' Team, good to see you here

This issue will helps readers in acquiring all the knowledge that one needs to know about **Types of Distribution**.

To get assigned to this issue, add your **Batch Numbers** mentioned in the spreadsheet of "Machine Learning", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

Domain : Machine Learning

Points to Note :

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title should be the same. Include issue number along with it.

- Changes should be made inside the `Machine Learning/Machine_Learning/Statistics_for_Machine_Learning` Branch.

- Follow Contributing Guidelines & Code of Conduct before start Contributing.

- This issue is only for 'GWOC' contributors of 'Machine Learning' domain.

**To be Mentioned while taking the issue :**
- Full name
- Batch Number
- GitHub Profile Link
- Which type of Contribution you want to make : 
    - [ ] Audio

All the best. Enjoy your open source journey ahead. 😎

### Domain

Machine Learning

### Type of Contribution

Audio

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by kishlayaug15** [comment]
I want  to contribute to this issue :
Full Name - Kishlay Kumar
Domain - Machine Learning
Batch - 4
Issue Number - 1963
github_ID - https://github.com/kishlayaug15
Type - Documentation, Audio

**Comment by abhisheks008** [comment]
What is the update @kishlayaug15?  It's been 3 days and you were although you've assigned an issue. Create the PR ASAP, otherwise we can't make further issues!

**Comment by kishlayaug15** [comment]
Yes! I am working on types of distribution almost done just give me a day.
Actually, can I create PR now or after completion of my work only I can
create it.
I have doubts related to it because I am new to open-source.

Thanks

**Comment by abhisheks008** [comment]
@kishlayaug15 If you've doubts ask your mentors of Batch 4, they will help you out and make the PR ASAP.

**Comment by abhisheks008** [comment]
Please do the needful and complete this issue @kishlayaug15

**Comment by kishlayaug15** [comment]
I completed the documentation and working on audio right now. Background
noises are problem. If you permit, I can commit my documenation right now.
For audio, I'll work tonight to get better sound quality.

Or any suggestion from your side regarding background noises will be
helpful.
